client/state/posts/selectors.js

```diff
@@ -359,6 +360,29 @@ export const isEditedPostDirty = createSelector(
 );
 
 /**
+  * Returns true if the edited post has content
+  * (title, excerpt or content not empty)
+  *
+  * @param  {Object}  state  Global state tree
+  * @param  {Number}  siteId Site ID
+  * @param  {Number}  postId Post ID
+  * @return {Boolean}        Whether the edited post has content or not
+ */
+export const editedPostHasContent = createSelector(
+	( state, siteId, postId ) => {
+		const editedPost = getEditedPost( state, siteId, postId );
+		return (
+			!! editedPost &&
+			(
+				some( [ 'title', 'excerpt' ], ( field ) => editedPost[ field ] && !! editedPost[ field ].trim() ) ||
+				! isPostContentEmpty( editedPost.content )
+			)
+		);
+	},
+	( state ) => [ state.posts.items, state.posts.edits ]
+);
```

Comment:
> how often does this get called? on every keystroke in the editor? on every dispatched action passed through the reducers?

It is called on every dispatch, for now the selector is used on only one place (one connect), but if we add it to a second component, it will be called twice on each dispatch.

> I guess I should just ask, how did you measure it?

I think those numbers are good. I used Google Chrome's profiler.

> why is not getEditedPost() the selector which is memoized

I agree, I think if we memoize `getEditedPost` we will have more gains because this one is called more often and from different selectors. IMO the two selectors can be memoized but let me run some tests again and will share the results.